modules/carousel/jetpack-carousel.js

```diff
@@ -948,16 +948,22 @@ jQuery(document).ready(function($) {
 
 			if ( 'undefined' === typeof args.medium_file || 'undefined' === typeof args.large_file ) {
 				return args.orig_file;
-			}
-
-			var medium_size       = args.medium_file.replace(/-([\d]+x[\d]+)\..+$/, '$1'),
-				medium_size_parts = (medium_size !== args.medium_file) ? medium_size.split('x') : [args.orig_width, 0],
+			}			
+			
+			// Check if the image is being served by Photon
+			
+			var imageLinkParser = document.createElement('a');
```

Comment:
While this file does not seem to follow the standard everywhere, let's try to make parts that we are changing follow it? Can you please add spaces inside braces where it applies?